picard/tagger.py

```diff
@@ -1246,12 +1246,52 @@ def sighandler(self):
         self.signalnotifier.setEnabled(True)
 
 
-def version():
-    print("%s %s %s" % (PICARD_ORG_NAME, PICARD_APP_NAME, PICARD_FANCY_VERSION_STR))
+class PicardArgs:
 
+    def __init__(self, argparse_args):
+        self.__load_from_argparse(argparse_args)
 
-def longversion():
-    print(versions.as_string())
```

Comment:
Why is this disappeared?

Reply:
Where is it used? I cannot find any usage